provider/schema/execution.go

```diff
@@ -228,10 +228,11 @@ func (e ExecutionData) callTableResolve(ctx context.Context, client ClientMeta,
 		if len(objects) == 0 {
 			continue
 		}
-		if err := e.resolveResources(ctx, client, parent, objects); err != nil {
+		resolvedCount, err := e.resolveResources(ctx, client, parent, objects)
+		if err != nil {
 			return 0, err
 		}
-		nc += uint64(len(objects))
+		nc += resolvedCount
```

Comment:
`insertCount`? not sure. the method is not called `insertResources` but `resolve` is something else (calling resolvers and gathering data) and not really related to data persistence?

Reply:
its total resources that were fully resolved i.e fetched + inserted.